src/notation/internal/notationactioncontroller.cpp

```diff
@@ -45,7 +44,7 @@ void NotationActionController::init()
     using Interaction = INotationInteraction;
 
     //! NOTE For historical reasons, the name of the action does not match what needs to be done
-    registerAction(ESCAPE_ACTION_CODE, &Controller::resetState, &Controller::isNotationPage);
+    registerAction("notation-escape", &Controller::resetState, &Controller::isNotationPage);
```

Comment:
NB this is a potentially breaking change for plugins that use cmd("escape") - I'm pretty sure at least one or two popular ones out there do use that.

Reply:
I added compat for renamed actions 
https://github.com/musescore/MuseScore/pull/10124/files#diff-8cb1084e2f3a9c1bf9cf91eb86b8da4a12806deb68794d63b65e27bf0b8b7ce7R216